packages/spor-react/src/input/PasswordInput.tsx

```diff
@@ -46,6 +46,7 @@ export const PasswordInput = forwardRef<PasswordInputProps, "input">(
             onClick={onToggle}
             borderRadius="sm"
             marginRight={1}
+            isDisabled={props.disabled || props.isDisabled}
```

Comment:
I forwarded the disabled props to the show/hide passwordInput toggle button, as it should be disabled when the input it self is disabled. Looks like PasswordInput allows both `disabled` and `isDisabled` as props, meaning both needs to be accounted for? How do you guys typically tackle this sitch?

Reply:
We should probably only allow `isDisabled`. I'll create another pull request that deprecates `disabled` in favor of `isDisabled`, and remove it in the next major